twilio/endpoint.bal

```diff
@@ -18,15 +18,23 @@ import ballerina/auth;
 import ballerina/http;
 import ballerina/mime;
 
-# Client for Twilio endpoint.
+# The Twilio API provides capability to access its platform for communications. These APIs connects 
+# the software layer and communications networks around the world to allow users to call and message anyone, globally. 
 #
 # + accountSId - Unique identifier of the account
 # + basicClient - HTTP client endpoint for basic api
 @display {label: "Twilio Client", iconPath: "logo.svg"}
-public client class Client {
-    public string accountSId;
-    public http:Client basicClient;
+public isolated client class Client {
+    private final string accountSId;
+    private final http:Client basicClient;
 
+    # Gets invoked to initialize the `connector`.
+    # The connector initialization requires setting the API credentials. 
+    # Create a [Twilio](https://www.twilio.com/) account and obtain account SID and auth token at [console](twilio.com/console) 
```

Comment:
Suggestion: "Create a [Twilio](https://www.twilio.com/) account and obtain account SID and auth token at [console](twilio.com/console). For more details, see [Twillio Documentation - Authenticate with HTTP](https://www.twilio.com/docs/usage/api#authenticate-with-http)."